dashboard/src/url/statechart.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name":"StateMachine",
```

Comment:
on the first glance seems fine, maybe this is enough and a hierarchical machine with healthy and unhealthy superstates are not needed. I am not sure.

But this is not accurate to the statemachine i have provided. Might still work fine, i will look at the rest of the code to see how it simplifies or complicates things to keep the sm flat